pysat/instruments/de2_lang.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""Supports the Langmuir Probe (LANG) instrument on
+Dynamics Explorer 2 (DE2).
+
+From CDAWeb:
+
+The Langmuir Probe Instrument (LANG) was a cylindrical electrostatic probe that
+obtained measurements of electron temperature, Te, and electron or ion
+concentration, Ne or Ni, respectively, and spacecraft potential.  Data from
+this investigation were used to provide temperature and density measurements
+along magnetic field lines related to thermal energy and particle flows within
+the magnetosphere-ionosphere system, to provide thermal plasma conditions for
+wave-particle interactions, and to measure large-scale and fine-structure
+ionospheric effects of energy deposition in the ionosphere.  The Langmuir Probe
+instrument was identical to that used on the AE satellites and the Pioneer
+Venus Orbiter. Two independent sensors were connected to individual adaptive
+sweep voltage circuits which continuously tracked the changing electron
+temperature and spacecraft potential, while autoranging electrometers adjusted
+their gain in response to the changing plasma density. The control signals used
+to achieve this automatic tracking provided a continuous monitor of the
+ionospheric parameters without telemetering each volt-ampere (V-I) curve.
+Furthermore, internal data storage circuits permitted high resolution, high
+data rate sampling of selected V-I curves for transmission to ground to verify
+or correct the inflight processed data. Time resolution was 0.5 seconds.
+
+
+References
+----------
+J. P. Krehbiel, L. H. Brace, R. F. Theis, W. H. Pinkus, and R. B. Kaplan,
+The Dynamics Explorer 2 Langmuir Probe (LANG), Space Sci. Instrum., v. 5, n. 4,
+p. 493, 1981.
+
+Parameters
+----------
+platform : string
+    Supports 'de2'
+name : string
+    Supports 'lang'
+sat_id : string
+    None Supported
+tag : string
+    None Supported
+
+Note
+----
+::
+
+    Notes
+
+Warnings
+--------
+
+
+Authors
+-------
+
+"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+
+import functools
+
+import pysat
+from .methods import nasa_cdaweb as cdw
```

Comment:
from pysat.methods is more robust.

Reply:
This is persistent throughout the cdaweb instruments.  Perhaps a separate pull?